testdata/serviceconfig.yml

```diff
@@ -17,16 +17,12 @@ rules:
         labels:
           source: source.name | "unknown"
           target: target.name | "unknown"
-          service: api.name | "unknown"
```

Comment:
We removed the `api.*` attributes from the canonical set, they're still up in the air. I'm fine keeping unknown, but I did go ahead and pull those fields out of the istio/istio repo's configs.